imgee/tasks.py

```diff
@@ -4,32 +4,58 @@
 
 class TaskRegistry(object):
     def __init__(self, name='default', connection=None):
-        self.connection = redis.from_url(connection) if connection else None
+        self.connection = connection or self.set_connection_from_url()
         self.name = name
-        self.key = 'imgee:registry:%s' % name
+        self.key_prefix = 'imgee:registry:%s' % name
 
-    def set_connection(self, connection=None):
-        connection = connection or app.config.get('REDIS_URL')
-        self.connection = redis.from_url(connection)
+    def set_connection_from_url(self, url=None):
+        url = url or app.config.get('REDIS_URL')
+        self.connection = redis.from_url(url)
+        return self.connection
 
-    def add(self, taskid):
-        self.connection.sadd(self.key, taskid)
-
-    def remove(self, taskid):
-        self.connection.srem(self.key, taskid)
-
-    def remove_all(self):
-        for k in self.get_all_keys():
-            self.remove(k)
+    def _make_key(self, taskid):
+        return "{key_prefix}:{taskid}".format(key_prefix=self.key_prefix, taskid=taskid)
```

Comment:
Single quotes and unicode string, as `.format` always enforces the type that it's being used with.